_community_videos/03_talk_series.md

```diff
@@ -0,0 +1,17 @@
+slug: pywhy-video
+layout: page
+description: >-
+  PyWhy Causality in Practice - Causal Representation Learning: Discovery of the Hidden World - Kun Zhang
+summary: >-
+  Causality is a fundamental notion in science, engineering, and even in machine learning. Causal representation learning aims to reveal the underlying high-level hidden causal variables and their relations. It can be seen as a special case of causal discovery, whose goal is to recover the underlying causal structure or causal model from observational data. The modularity property of a causal system implies properties of minimal changes and independent changes of causal representations, and in this talk, we show how such properties make it possible to recover the underlying causal representations from observational data with identifiability guarantees: under appropriate assumptions, the learned representations are consistent with the underlying causal process. Various problem settings are considered, involving independent and identically distributed (i.i.d.) data, temporal data, or data with distribution shift as input. We demonstrate when identifiable causal representation learning can benefit from flexible deep learning and when suitable parametric assumptions have to be imposed on the causal process, with various examples and applications.
+  <br>
+  The talk will include a description of the causal-learn package in PyWhy. Learn more: https://github.com/py-why/causal-learn
+  <br>
+  Speaker: Kun Zhang is currently on leave from Carnegie Mellon University (CMU), where he is an associate professor of philosophy and an affiliate faculty in the machine learning department; he is working as a professor and the acting chair of the machine learning department and the director of the Center for Integrative AI at Mohamed bin Zayed University of Artificial Intelligence (MBZUAI). He develops methods for making causality transparent by torturing various kinds of data and investigates machine learning problems including transfer learning, representation learning, and reinforcement learning from a causal perspective. He has been frequently serving as a senior area chair, area chair, or senior program committee member for major conferences in machine learning or artificial intelligence, including UAI, NeurIPS, ICML, IJCAI, AISTATS, and ICLR.  He was a general & program co-chair of the first Conference on Causal Learning and Reasoning (CLeaR 2022), a program co-chair of the 38th Conference on Uncertainty in Artificial Intelligence (UAI 2022), and is a general co-chair of UAI 2023.
+  <br>
+
+  <b><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_N2E0NzAxOTctMDAxNC00ZTY2LWE5ODYtZDU5YjhmNmRlZmM4%40thread.v2/0?context=%7b%22Tid%22%3a%22492f1487-e76c-454d-aad1-28c9aaf849f3%22%2c%22Oid%22%3a%22404ab0c2-59ec-4b36-88e9-81f01946470f%22%7d">Join the live seminar</a> on January 29, 2024 at Monday 8:00am pacific / 11:00am eastern / 4:00pm GMT / 9:30pm IST.
```

Comment:
``` suggestion 
  <b><a href="https://teams.microsoft.com/l/meetup-join/19%3ameeting_N2E0NzAxOTctMDAxNC00ZTY2LWE5ODYtZDU5YjhmNmRlZmM4%40thread.v2/0?context=%7b%22Tid%22%3a%22492f1487-e76c-454d-aad1-28c9aaf849f3%22%2c%22Oid%22%3a%22404ab0c2-59ec-4b36-88e9-81f01946470f%22%7d">Join the live seminar</a> on January 29, 2024 at Monday 8:00am pacific / 11:00am eastern / 4:00pm GMT / 9:30pm IST.</b>
```

This needs to be added to avoid a weird display issue